### DISCUSSION TITLE
Cannot read properties of undefined (reading 'places')

### DISCUSSION THREAD

**Discussion by skillcraft-io** [vueform/vueform#361] [Questions]
Hi, 

Does anyone have a working example of using the location element; 

Specifically with Laravel.
When I include the script, I get a ```InvalidValueError: initMap is not a function```

Thanks

**Comment by adamberecz** [comment]
Here: https://vueform.com/reference/location-element#prerequisites

The link was indeed pointing to the wrong tag (fixed), so this is the right one: https://vueform.com/reference/location-element#google
